src/status_im/multiaccounts/login/core.cljs

```diff
@@ -43,6 +43,7 @@
     [status-im2.contexts.contacts.events :as contacts]
     [status-im2.navigation.events :as navigation]
     [status-im2.contexts.shell.constants :as shell.constants]
+    [status-im2.contexts.communities.discover.events :as contract-communities]
```

Comment:
looks confusing, better to just use the part of the namespace for alias, smth like `communities.discover` in that case its more clear

Reply:
Related to [comment above](https://github.com/status-im/status-mobile/pull/16232#discussion_r1235323309). `contract-communities`, IMO, clearly identifies it as smart contract data (which was a source of significant confusion earlier), regardless of whether we use it in a feature called `discover`, or any other feature.